tests/test_edgeql_ddl.py

```diff
@@ -8898,7 +8896,10 @@ async def test_edgeql_ddl_enum_05(self):
                  CREATE CONSTRAINT expression ON (
                      <str>.num != asdf2()
                  );
-                 CREATE INDEX ON (asdf(.color));
+                 # FIXME: An index on asdf(.color) isn't supported currently
+                 # because passing nullable arguments requires filtering.
+                 # CREATE INDEX ON (asdf(.color));
+                 CREATE INDEX ON (.color);
```

Comment:
Unless you want to fix this as part of this PR, I'd create an x-fail test so that we don't forget about this.

Reply:
I just fixed it instead